src/openrct2-ui/windows/Research.cpp

```diff
@@ -362,16 +363,33 @@ void window_research_development_page_paint(rct_window* w, rct_drawpixelinfo* dp
     else
     {
         // Research type
-        stringId = STR_RESEARCH_UNKNOWN;
+        std::array<rct_string_id, 2> strings = { STR_RESEARCH_UNKNOWN, 0 };
```

Comment:
Shouldn't we be using the Formatter here instead of a std::array?

Reply:
`strings` acts as a buffer here, not as a formatter.

The formatter would still have to format to _a_ buffer - either this one or gCommonFormatArgs. But I really don't want to introduce any more instances of the latter.